BreweryRatings/Models/BusinessReview.cs

```diff
@@ -17,50 +9,66 @@ namespace ReviewNamespace
 
     public partial class BusinessReview
     {
+        private Review[] reviews;
+        private long total;
+        private string[] possibleLanguages;
+
         [JsonProperty("reviews")]
-        public Review[] Reviews { get; set; }
+        public Review[] Reviews { get => reviews; set => reviews = value; }
```

Comment:
Either way is fine... while this change is proper encapsulation, .net considers {get; set;} to be shorthand for the longer form.